test/feature/features/memqdb.feature

```diff
@@ -60,7 +88,81 @@ Feature: MemQDB save state into a file
     []
     """
 
+  Scenario: Sharding rules initilized on startup without backups
+    Given cluster environment is
+    """
+    ROUTER_CONFIG=/spqr/test/feature/conf/router_with_initsql.yaml
+    """
+    Given cluster is up and running
+    When I run SQL on host "spqr-console"
+    """
+    SHOW sharding_rules;
+    """
+    Then command return code should be "0"
+    And SQL result should match json_exactly
+    """
+    [
+      {
+          "Columns":"id",
+          "Hash Function":"x->x",
+          "Sharding Rule ID":"rule1",
+          "Table Name":"*"
+      },
+      {
+          "Columns":"idx",  
+          "Hash Function":"x->x",
+          "Sharding Rule ID":"rule2",
+          "Table Name":"test"
+      },
+      {
+          "Columns":"idy",
+          "Hash Function":"x->x",
+          "Sharding Rule ID":"rule3",
+          "Table Name":"*"
+      }
+    ]
+    """
+
+  Scenario: Sharding rules initilized on startup even with backups
+    Given cluster environment is
+    """
+    ROUTER_CONFIG=/spqr/test/feature/conf/router_with_backup_and_initsql.yaml
+    """
+    Given cluster is up and running
+    When I run SQL on host "spqr-console"
+    """
+    SHOW sharding_rules;
+    """
+    Then command return code should be "0"
+    And SQL result should match json_exactly
+    """
+    [
+      {
+          "Columns":"id",
+          "Hash Function":"x->x",
+          "Sharding Rule ID":"rule1",
+          "Table Name":"*"
+      },
+      {
+          "Columns":"idx",  
+          "Hash Function":"x->x",
+          "Sharding Rule ID":"rule2",
+          "Table Name":"test"
+      },
+      {
+          "Columns":"idy",
+          "Hash Function":"x->x",
+          "Sharding Rule ID":"rule3",
+          "Table Name":"*"
+      }
+    ]
+    """
```

Comment:
It would be cool to modify rules via spqr-console here, restart router and ensure that backup file works

Reply:
And what happen if backup file and Init SQL is specified but files does not exists?

Reply:
> It would be cool to modify rules via spqr-console here, restart router and ensure that backup file works

I did it at `Scenario: Sharding rules restored after droping specific one (init.sql ignored)`

Reply:
> And what happen if backup file and Init SQL is specified but files does not exists?

I will add test with `Scenario: Sharding rules not initialized if `init.sql` file doesn't exists`

Backup file don't exists in all scenarios, so this case already tested

Reply:
`Scenario: Sharding rules not initialized if init.sql file doesn't exists`

it turned out that router failed to start in this scenario. Maybe it make scene?